climada/hazard/tc_tracks.py

```diff
@@ -2890,6 +2896,10 @@ def compute_track_density(
     density: bool (optional) default: False
         If False it returns the number of samples in each bin. If True, returns the
         probability density function at each bin computed as count_bin / tot_count.
+    filter_tracks: bool (optional) default: True
```

Comment:
perhaps calling this argument something like `count_tracks` would be more explicit?

Reply:
I agree `filter_track` it's a bit cryptic...